base/applications/rapps/loaddlg.cpp

```diff
@@ -430,732 +432,736 @@ MessageBox_LoadString(HWND hOwnerWnd, INT StringID)
     }
 }
 
+static BOOL
+ShowLastError(HWND hWndOwner, BOOL bInetError, DWORD dwLastError)
+{
+    CLocalPtr<WCHAR> lpMsg;
+
+    if (!FormatMessageW(
+            FORMAT_MESSAGE_ALLOCATE_BUFFER | FORMAT_MESSAGE_IGNORE_INSERTS |
+                (bInetError ? FORMAT_MESSAGE_FROM_HMODULE : FORMAT_MESSAGE_FROM_SYSTEM),
+            (bInetError ? GetModuleHandleW(L"wininet.dll") : NULL), dwLastError, LANG_USER_DEFAULT, (LPWSTR)&lpMsg, 0,
+            NULL))
```

Comment:
minor formatting nit
```suggestion
            (bInetError ? GetModuleHandleW(L"wininet.dll") : NULL), dwLastError, LANG_USER_DEFAULT,
            (LPWSTR)&lpMsg, 0, NULL))
```